app/controllers/home_controller.rb

```diff
@@ -15,4 +15,13 @@ def destroy
       render :index
     end
   end
+
+  def send_mail
```

Comment:
Consider defining a separate controller and using the REST pattern to deliver the functionality.